homeassistant/components/syncthru/__init__.py

```diff
@@ -60,12 +75,12 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
-def device_identifiers(printer: SyncThru) -> set[tuple[str, ...]]:
+def device_identifiers(printer: SyncThru):
```

Comment:
Please type the signature completely, including return value, if typing any part of the signature.

Reply:
Ok. I will remove the signature completely since the current setup forbids me to set the correct signature The reason is that `set` and `tuple` (lowercase), cannot be typed, however some tool in the pre-commit chain before that rewrites `Set` and `Tuple` to their lowercase correspondences.

Reply:
`set` and `tuple` can be used as type annotations.